apps/api/src/city/city.controller.ts

```diff
@@ -12,4 +13,28 @@ export class CityController {
   async getData() {
     return await this.cityService.listCities()
   }
+
+  @Get('list-one/:id')
+  @Public()
+  findOne(@Param('id') id: string) {
+    return this.cityService.listCity(id)
+  }
+
+  @Post('create-city')
+  @Public()
+  async create(@Body() CreateCityDto: CreateCityDto) {
+    return await this.cityService.createCity(CreateCityDto)
+  }
+
+  @Patch('update-one/:id')
```

Comment:
Please remove the `update-one/` from the path

```suggestion
  @Patch(':id')
```